client/web/src/enterprise/insights/core/hooks/live-preview-insight/use-live-preview-series-insight.ts

```diff
@@ -92,6 +93,16 @@ export function useLivePreviewSeriesInsight(props: Props): Result<Series<Datum>[
     }
 
     if (error) {
+        if (isGatewayTimeoutError(error)) {
+            return {
+                state: {
+                    status: LivePreviewStatus.Error,
+                    error: new Error('Live preview is not available for this chart as it will take longer to generate'),
```

Comment:
```suggestion
                    error: new Error('Live preview is not available for this chart as it did not complete in the allowed time'),
```
I don't know if this is any better, having trouble finding the right words to briefly describe the timeout

Reply:
I think your version is better because it says that we tried but we failed rather than we know that this will fail that we won't even try (which isn't correct because we did try)